### ISSUE TITLE
command test 3516 fails on some platforms

### ISSUE THREAD

**Issue by jgm** [jgm/pandoc#3526]
We're getting a test failure on appveyor (Windows) which we don't see on linux or osx.
https://ci.appveyor.com/project/jgm/pandoc/build/1.0.996

Can anyone else who builds on Windows reproduce this?

The failure results from slightly larger table cells in grid tables.
We've confirmed that the parse results are the same, so this concerns the RST writer (and similarly the Markdown writer, which shares this code).

**Comment by jgm** [comment]
Apparently this is not windows related.  We have the same thing reported on kubuntu.
https://groups.google.com/d/msgid/pandoc-discuss/87bmst8c8r.fsf%40mat.ucm.es

**Comment by jgm** [comment]
We're getting this result:
```
+---+---+
| 1 | 2 |
+---+---+
|  |  |
+---+---+
```

**Comment by jgm** [comment]
I couldn't reproduce the problem on a debian jessie machine.
Please, if anyone can reproduce, please give details of your setup (pandoc version, how installed, architecture, anything unusual).  To reproduce, run
```
pandoc -t rst
```
on the following input:
```
+---+---+
| 1 | 2 |
+---+---+
|   |   |
+---+---+
```
If the cells are not even in width, it's the bug.

**Comment by jgm** [comment]
Possible clue, which I can reproduce on my machine:
```
% pandoc -t rst
+--+---+
|  |   |
+--+---+
|  |   |
+--+---+
^D
+---+---+
|  |   |
+---+---+
|  |   |
+---+---+
```
note: widthInChars is [0,1].

**Comment by jgm** [comment]
This seems to have been fixed by the above commit a939cfe.
